Assets/Resources/Category.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
```

Comment:
I think this and the other .cs files should be moved the src/QuestionStore folder

Reply:
Good call, not sure why I left them there.